scripts/build_and_publish_docker_images.sh

```diff
@@ -4,19 +4,20 @@ set -e
 # If not staging and master branch are existing
 export TAG="$CIRCLE_BRANCH"
 
-if [ "$CIRCLE_BRANCH" == "staging" ]; then 
+if [ "$CIRCLE_BRANCH" == "staging" ]; then
     export TAG="latest"
+    cp .env.latest .env
     echo $TAG
 fi
 
 if [ "$CIRCLE_BRANCH" == "master" ]; then
-    export TAG="stable"; 
+    export TAG="stable";
+    cp .env.build .env
```

Comment:
@edisinovcic Do you think this should this be `build` or `stable`? At the moment staging is published with build.

Reply:
`cp .env.${TAG} .env
should fix it